website/content/api-docs/system/internal-counters.mdx

```diff
@@ -1088,8 +1088,12 @@ months in the requested time range.
 information returned by this API will include activity for this month, however
 it may be up to 20 minutes delayed.
 
+- This endpoint can be called from all namespaces. The requested namespace (specified via the `X-Vault-Namespace`
+  header or as part of the path) will act as a filter. The exported data will include activity for the requested
+  namespace and all of its children.
```

Comment:
```suggestion
- This endpoint can be called from all namespaces. The requested namespace will act as a filter. The exported data will include activity for the requested
  namespace and all of its children.
```

Reply:
There are other ways for namespaces to be selected, such as the chroot namespace listener functionality -- I'd rather not enumerate them all here and we may add more in the future

Reply:
That's a great point, thanks for the feedback!